docs/data-sources/repository_credential.md

```diff
@@ -0,0 +1,153 @@
+---
+Title: "Repository Credential Data Source"
+Description: |-
+    Creating the Repository Credential data source.
+---
+
+# Repository Credential
+
+The `tanzu-mission-control_repository_credential` data source allows you to get repository credential to a particular scope through Tanzu Mission Control.
```

Comment:
Again, kindly update a rich description to describe the resource in general.

Reply:
Updated PTAl

Reply:
I don't see the updates.